docs/DevGuide/Notation.md

```diff
@@ -0,0 +1,127 @@
+  \f$(-,+,+,+)\f$, and its determinant by \f$g\f$.
+- The lapse and shift are denoted by \f$\alpha\f$ and \f$\beta^i\f$,
+  respectively.
+- The spatial metric and its determinant are denoted by \f$\gamma_{ij}\f$ and
+  \f$\gamma\f$, respectively.
+- We write the line element as:
+  \f{align*}{
+      ds^2=g_{ab}dx^a dx^b=-\alpha^2 dt^2 +
+      \gamma_{ij}
+      \left(dx^i + \beta^i dt\right)\left(dx^j + \beta^j dt\right).
+  \f}
+- The unit timelike normal to the hypersurface is
+  \f$n^a=(1/\alpha,-\beta^i/\alpha)\f$ and \f$n_a=(-\alpha,0,0,0)\f$.
+- The extrinsic curvature is denoted by \f$K_{ij}\f$ and its trace by
+  \f$K\f$. We use the sign convention
+  \f{align*}{
+      K_{ab}=-\frac{1}{2}\mathcal{L}_n g_{ab}
+  \f}
+- For the generalized harmonic evolution system \f$\Pi_{ab}=-n^c\partial_c
+  g_{ab}\f$ and \f$\Phi_{iab}=\partial_i g_{ab}\f$.
+
+The notation for (magneto)hydrodynamics used is:
+- The fluid pressure, rest mass density, specific internal energy, and specific
+  enthalpy are denoted by \f$p\f$, \f$\rho\f$, \f$\epsilon\f$, and \f$h\f$,
+  respectively.
+- The 4-velocity of the fluid is \f$u^a\f$ and \f$u^a u_a=-1\f$.
+- The spatial velocity of the fluid as measured by an observer at rest in the
+  spatial hypersurfaces (“Eulerian observer”) is
+  \f{align*}{
+      v^i=\frac{1}{\alpha}\left(\frac{u^i}{u^0}+\beta^i\right)
+  \f}
+  with corresponding Lorentz factor
+  \f{align*}{
+      W=-u^a n_a=\alpha u^0=\frac{1}{\sqrt{1-\gamma_{ij}v^i v^j}}
+  \f}
+- The Faraday tensor is denoted by \f$F^{ab}\f$ and its dual is given by
+  \f$^{*}\!F^{ab}=\frac{1}{2}\epsilon^{abcd}F_{cd}\f$ where
+  \f$\epsilon^{abcd}\f$ is the Levi-Civita tensor.
+- The magnetic field in the frame comoving with the fluid is
+  \f{align*}{
+      b^a = ^{*}\!F^{ab}u_b
+  \f}
+- The electric and magnetic fields as measured by an Eulerian observer are given
+  by
+  \f{align*}{
+      E^i=&F^{ia}n_a=\alpha F^{0i} \\
+      B^i=&^{*}\!F^{ia}n_a=\alpha^{*}\!F^{0i}.
+  \f}
+- \f$b^a\f$ can be written in terms of \f$E^i\f$ and \f$B^i\f$ as follows:
+  \f{align*}{
+      b^0=&\frac{W}{\alpha}B^i v_i\\
+      b^i=&\frac{B^i+\alpha b^0u^i}{W}
+  \f}
+  while \f$b^2=b^i b_i\f$ is given by
+  \f{align*}{
+      b^2=\frac{B^2}{W}+\left(B^i v_i\right)
+  \f}
+- The magnetic pressure is \f$p_{\mathrm{mag}}=b^2/2\f$
+- The primitive variables are \f$\rho\f$, \f$\epsilon\f$, \f$v_i\f$, \f$B^i\f$,
+  and the divergence cleaning field \f$\Phi\f$.
+- The conserved variables are:
+  \f{align*}{
+      D=&\rho W\\
+      S_i=&(\rho h)^{*}W^2 v_i - \alpha b^0 b_i \\
+      \tau =& (\rho h)^{*}W^2 - p^* - \left(\alpha b^0\right)^2-\rho W \\
+      B^i =& B^i \\
+      \Phi =& \Phi,
+  \f}
+  where
+  \f{align*}{
+      (\rho h)^* =& \rho h + b^2=\rho h + 2 p_{\mathrm{mag}} \\
+      p^* = & p + \frac{b^2}{2}=p+p_{\mathrm{mag}}.
+  \f}
+  We denote the conserved variables multiplied by \f$\sqrt{\gamma}\f$ with a
+  tilde on top. For example, \f$\tilde{D}=\sqrt{\gamma}D\f$.
```

Comment:
Yes, but I was hoping @markscheel could do this since he's most familiar with them. I'm trying to get what is in the current paper draft into the code.